### PR TITLE
Script for splitting data for cross-validation

### PR DESCRIPTION
**Pull request by mirunacrt**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -56,3 +58,6 @@ dev =
 
 [flake8]
 extend-ignore = E203, E501
+
+[mypy-sklearn.*]
+ignore_missing_imports = True
```
Comment (avaucher) [inline setup.cfg:63]: Remove here and add in `pyproject.toml` instead, in the list that already exists (along with `rdkit`)

File: src/rxn_standardization/utils.py [hunk 2]
```diff
@@ -84,3 +86,25 @@ def get_sequence_multiplier(ground_truth: Sequence[T], predictions: Sequence[T])
     n_pred = len(predictions)
 
     return get_multiplier(n_gt, n_pred)
+
+
+def augment(
+    original_smiles: List,
```
Comment (avaucher) [inline src/rxn_standardization/utils.py:92]:
```suggestion
    original_smiles: List[str],
```

File: src/rxn_standardization/utils.py [hunk 3]
```diff
@@ -84,3 +86,25 @@ def get_sequence_multiplier(ground_truth: Sequence[T], predictions: Sequence[T])
...
+    detokenize: bool,
+) -> List:
```
Comment (avaucher) [inline src/rxn_standardization/utils.py:94]:
```suggestion
) -> List[str]:
```

File: README.md [hunk 4]
```diff
@@ -43,6 +43,12 @@ rxn-std-process-csv --help
 src-test.txt    src-train.txt   src-valid.txt   tgt-test.txt    tgt-train.txt   tgt-valid.txt
 ```
 
+To perform multiple data splits for cross-validation, run:
+```bash
+rxn-std-split-for-cv --help
+```
+and follow the instructions.
```
Comment (avaucher) [inline README.md:50]: follow which instructions? 

Can you be a bit more specific here?

File: src/rxn_standardization/scripts/split_for_cv.py [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+    save_dir: str,
+    test_size: int,
+    prepend_token: Optional[str],
+    augmentation: bool,
+    augment_for_tautomers: bool,
+):
+    setup_console_logger()
+
+    all_smiles = load_list_from_file(input_csv)
+    random.shuffle(all_smiles)
```
[the remark this answers is not in the document]
Comment (mirunacrt) [reply src/rxn_standardization/scripts/split_for_cv.py:81]: yes, this assumes it is discarded. Should I add `all_smiles = load_list_from_file(input_csv)[1:]`?

[the remark this answers is not in the document]
Comment (avaucher) [reply src/rxn_standardization/scripts/split_for_cv.py]: yes 👍
